marklogic-data-hub/src/main/resources/hub-internal-artifacts/step-definitions/mapping/marklogic/entity-services-mapping.step.json

```diff
@@ -15,7 +15,7 @@
       "default-mapping",
       "mdm-content"
     ],
-    "validateEntity": false,
+    "validateEntity": "notValidate",
```

Comment:
I am not sure if "notValidate" sounds good for the value of this property. 
@ryanjdew , @rjrudin, 
Is this value good ? , if not, any suggestions for the value for this property ? How does "none" sound like ?

Reply:
"doNotValidate" would be more closely aligned with UI